### PR TITLE
annotate api with @Nullable

### PR DESCRIPTION
**Pull request by vietj**
the api has been annotated with `@Nullable` where it makes sense to improve the polyglot API.

please review

### CODE REVIEW

File: src/main/java/io/vertx/core/cli/CLI.java [hunk 1]
```diff
@@ -88,6 +89,7 @@ static CLI create(Class<?> clazz) {
   /**
    * @return the CLI name.
    */
+  @Nullable
```
Comment (cescoffier) [inline src/main/java/io/vertx/core/cli/CLI.java]: That's a good question. I don't think the name can be null.

**Comment by cescoffier** [comment]
I've a doubt about the CLI name. Everything else looks good.

File: src/main/java/io/vertx/core/cli/CLI.java (hunk 1 above)
Comment (cescoffier) [inline src/main/java/io/vertx/core/cli/CLI.java]: The name of the CLI cannot be null.

Comment (vietj) [inline src/main/java/io/vertx/core/cli/CLI.java]: ok :-)

**Pull request merged**